src/Elliptic/Systems/Elasticity/Equations.hpp

```diff
@@ -94,7 +94,9 @@ struct Sources {
   using argument_tags = tmpl::list<>;
   static void apply(
       const gsl::not_null<tnsr::I<DataVector, Dim>*> source_for_displacement,
-      const tnsr::I<DataVector, Dim>& /*displacement*/) noexcept {
+      const gsl::not_null<tnsr::ii<DataVector, Dim>*> /*source_for_strain*/,
+      const tnsr::I<DataVector, Dim>& /*displacement*/,
```

Comment:
aren't the new arguments fluxes? Maybe make that clear in the name?

Reply:
- The new `source_for_strain` argument is the source for the auxiliary equation (think the equation grad(u) = v that defines the auxiliary variable for a Poisson equation. The auxiliary variable for the elasticity system is the symmetric "strain", i.e. the symmetrized gradient of the displacement vector). The source computers gain this argument to get the opportunity to add non-principal contributions to the gradient (Christoffel-symbol terms) to the auxiliary equation.
- The new `stress` argument is indeed the flux related to the displacement vector field. It's the strain contracted with the constitutive relation of the elastic material, and that's called "stress" in elasticity terms. I think in the context of the elasticity equations it makes sense to call it by its name.

Reply:
Okay, I can't speak to what would be most intuitive for people used to the elasticity literature, so I'll have to take your word for it :)